mavros_tf_publisher:
  world_frame: "map"
  drone_frame: "base_link"